### ISSUE TITLE
zipkin trace has fixed dep

### ISSUE THREAD

**Issue by miekg** [coredns/coredns#1193]
zipkin tracing is stuck to v0.2.3
~~~
[[constraint]]
    name = "github.com/openzipkin/zipkin-go-opentracing"
    version = "v0.2.3"
~~~
We should probably not pin deps like this and follow upstream for as much stuff as possible.

**Comment by yongtang** [comment]
The `zipkin-go-opentracing` `v0.2.3` is almost one year old and the `thrift` it depends on is one year old as well. I probably will feel more comfortable with a newer commit. Let me see what we could do here.

**Comment by yongtang** [comment]
Interesting. The `zipkin-go-opentracing ` `v0.3.1` is not compatible with the head of the `thrift` as well. And I could not find the version of the `thrift` that is being used by `zipkin-go-opentracing` when they release  `v0.3.1`...

**Comment by yongtang** [comment]
The issue is listed in https://github.com/openzipkin/zipkin-go-opentracing/issues/68 and https://github.com/openzipkin/zipkin-go-opentracing/issues/86 as well.

Looks like it has been fixed recently by PR https://github.com/openzipkin/zipkin-go-opentracing/pull/90

**Comment by basvanbeek** [comment]
sorry about the late fix guys, I was traveling when thrift made a breaking change again. just released v0.3.2 which includes fix to latest thrift go library master.

**Comment by yongtang** [comment]
Thanks @basvanbeek for the fix! 👍
